deployment/config.go

```diff
@@ -18,68 +15,53 @@ import (
 // Config contains the necessary data
 // to deploy and provision a load test environment.
 type Config struct {
-	ClusterName         string // Name of the cluster.
-	AppInstanceCount    int    // Number of application instances.
-	AppInstanceType     string // Type of the EC2 instance for app.
-	AgentInstanceCount  int    // Number of agents, first agent and coordinator will share the same instance.
-	AgentInstanceType   string // Type of the EC2 instance for agent.
-	EnableAgentFullLogs bool   // Logs the command output (stdout & stderr) to home directory.
-	ProxyInstanceType   string // Type of the EC2 instance for proxy.
-	SSHPublicKey        string // Path to the SSH public key.
-	DBInstanceCount     int    // Number of DB instances.
-	DBInstanceType      string // Type of the DB instance.
-	DBInstanceEngine    string // Type of the DB instance - postgres or mysql.
-	DBUserName          string // Username to connect to the DB.
-	DBPassword          string // Password to connect to the DB.
+	// ClusterName is the name of the cluster.
+	ClusterName string `default:"loadtest" validate:"alpha"`
+	// Number of application instances.
+	AppInstanceCount int `default:"1" validate:"range:[1,)"`
+	// Type of the EC2 instance for app.
+	AppInstanceType string `default:"c5.xlarge" validate:"oneof:{c5.xlarge, t3.xlarge, m4.xlarge}"`
```

Comment:
I wouldn't want to use t3 or m4 for apps. Just have all the types in c5 category.